Fisticuffs.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   s.author           = { "Darren Clark" => "darren.clark@thescore.com" }
   s.source           = { :git => "https://github.com/scoremedia/Fisticuffs.git", :tag => s.version.to_s }
 
-  s.platform     = :ios, '8.0'
+  s.platform     = :ios, '10.0'
```

Comment:
I'm wondering if we should keep this at 8.0

Reply:
Maybe, I don't think theres anything new that would prevent it from being used on iOS 8